go.mod

```diff
@@ -7,22 +7,33 @@ toolchain go1.23.7
 require (
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/calindra/rollups-base-reader v0.0.0-20250311174618-9bed2f04dd33
+	github.com/calindra/rollups-base-reader v0.0.0-20250404035155-73d1262bb8dc
```

Comment:
I added a task for us to create a tag.
https://github.com/orgs/cartesi/projects/12?pane=issue&itemId=105051805